### DISCUSSION TITLE
dfinity-sdk: command failed

### DISCUSSION THREAD

**Discussion by Jaseempk** [dfinity/sdk#2811] [Unused 2]
hey I'm trying to add dfx into my wsl ubuntu, every time I use the command ` DFX_VERSION=0.11.1 sh -ci "$(curl -fsSL https://smartcontracts.org/install.sh)"` I'm getting the error as shown in the pic below:
![Screenshot 2022-12-10 203956](https://user-images.githubusercontent.com/99121494/206862049-bfd5ceec-6ed8-4dc9-89f3-2122fa7646fe.png)
can anyone please hep me?

**Comment by viviveevee** [comment]
Locking this as duplicate of https://github.com/dfinity/sdk/issues/2501
